engine/src/org/pentaho/di/trans/steps/textfileoutput/TextFileOutput.java

```diff
@@ -647,7 +647,7 @@ public void openNewFile( String baseFilename ) throws KettleException {
         }
 
         if ( log.isDetailed() ) {
-          logDetailed( "Opened new file with name [" + filename + "]" );
+          logDetailed( "Opened new file with name [" + KettleVFS.getFriendlyURI( filename ) + "]" );
```

Comment:
Check the changes in this file to see if any of the getFriendlyURI calls should be using the FileObject signature instead.